.github/workflows/ci-cd.yaml

```diff
@@ -0,0 +1,29 @@
+name: ci-cd
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: ./bin/hermit env -r >> $GITHUB_ENV
+    - run: make
+
+  release:
+    runs-on: ubuntu-latest
+    needs: [ci]
+    if: github.event_name == 'push' # only run on push to master
+    steps:
+    - uses: actions/checkout@v2
+    - run: git fetch --prune --unshallow --tags
+    - run: ./bin/hermit env -r >> $GITHUB_ENV
+    - run: make release
+    - if: ${{ failure() }}
+      run: ./slack-notify
```

Comment:
should be `./slack-notify.sh` (added extension)

Reply:
yes, oups... i think i should add this to reference itself, i just want to get this PR in first i think and tag it.

Reply:
as is it should be `uses: foxgoat/howl`